data/tools/unit_tree/html_output.py

```diff
@@ -583,9 +587,9 @@ def end_menu(is_table_container=False):
             target = "mainline.html"
 
         if not self.is_era:
-            add_menu("races_menu", _("Race", "wesnoth-lib"))
+            add_menu("races_menu", _w_lib("Race"))
 
-            add_menuitem('mainline.html', _("all", "wesnoth-editor"))
+            add_menuitem('mainline.html', _w_editor("all"))
```

Comment:
Not an issue for this PR, but this is the only string coming from `_w_editor`, it's got no context to say what it is, and it seems out of place.

The meaning here is "show the page with the full list of units sorted by race, and jump to the top", in a menu where every other option is the name of a race, but just means "jump to the same unfiltered list, but scroll to the race's units". So if we weren't constrained to available strings, I'd suggest "top" or "top of page".

The meaning in `wesnoth-editor` is "show an unfiltered list of terrains/units/items". The same meaning in `wesnoth-lib` is found in the add-on manager's "None Selected", which wouldn't fit here.